.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: python
 python:
-- '3.5'
+  - '3.9'
```

Comment:
It's possible to specify a subversion of 3.9 like 3.9.1 but for now nothing above 3.9.1 is available. Using just 3..9 like this will make it use the most recent available version, the exact version is printed out at the start of every travis build.